src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -146,14 +150,18 @@ TernaryRadios.propTypes = {
   formResponses: PropTypes.object.isRequired,
   h1: PropTypes.string.isRequired,
   responses: PropTypes.arrayOf(PropTypes.string).isRequired,
-  router: PropTypes.object.isRequired,
+  router: PropTypes.shape({
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.